portfolio/src/main/webapp/index.html

```diff
@@ -48,6 +62,8 @@ <h2>Some Projects I Worked On</h2>
        <a href="images/proj2.png"><img src="images/proj2.png"/></a>
 </body>
 
+<p id="elementcontainer"></p>
```

Comment:
See https://google.github.io/styleguide/htmlcssguide.html#id_Attributes for naming id attributes, but instead of elementcontainer, 1) use of hyphen is recommended and 2) I would try to name it something more descriptive like "greeting-container".